index.js

```diff
@@ -216,7 +216,7 @@ async function checkHeartbeat() {
         sendReconnectionMessage(location.locationid, location.displayName)
       }
     } catch (err) {
-      helpers.logError(`Error checking heartbeat: ${err}`)
```

Comment:
These ones that are just printing `${err}` should be find. I think it's only the `${JOSN.stringify(err)}` ones that need to change.